install/python-installer/scripts/traf_ldap.py

```diff
@@ -34,8 +34,9 @@ def run():
 
     db_root_user = dbcfgs['db_root_user']
     traf_home = os.environ['TRAF_HOME']
+    traf_conf = os.environ['TRAF_CONF']
     sqenv_file = traf_home + '/sqenvcom.sh'
-    traf_auth_config = '%s/sql/scripts/.traf_authentication_config' % traf_home
+    traf_auth_config = '%s/.traf_authentication_config' % traf_conf
```

Comment:
Should we remove the `.` from the ldap config file? Otherwise it will be hided in $TRAF_CONF dir if not provide `ls -a` command. I'm curious why we hide this config file in the original design, maybe it's time to change it.

Reply:
Yes. Unrelated to this change, but I agree that it is strange we name the file that way.